edgePy/data_import/mongodb/mongo_wrapper.py

```diff
@@ -266,6 +306,8 @@ def add(self, updatedict: Dict[Hashable, Any], setdict: Dict[Hashable, Any]) ->
     def close(self) -> None:
         """
         Close the MongoInserter - flush the buffer.
-        :return: None
+
+        Return:
+            None
```

Comment:
Don't think it needs the Return if it's not returning anything. https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings